Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -238,7 +299,7 @@ extension VisualInstruction.Component: Codable {
                 imageBaseURL = URL(string: imageBaseURLString)
             }
             let imageRepresentation = ImageRepresentation(imageBaseURL: imageBaseURL)
-            self = .image(image: imageRepresentation, alternativeText: textRepresentation)
+            self = .image(image: imageRepresentation, alternativeText: textRepresentation, mapboxShield: mapboxShield)
```

Comment:
The changelog should mention that this enumeration case has been renamed from `VisualInstruction.Component.